src/error_gui.cpp

```diff
@@ -175,11 +176,13 @@ struct ErrmsgWindow : public Window, ErrorMessageData {
 private:
 	uint height_summary;            ///< Height of the #summary_msg string in pixels in the #WID_EM_MESSAGE widget.
 	uint height_detailed;           ///< Height of the #detailed_msg string in pixels in the #WID_EM_MESSAGE widget.
+	GUITimer duration_timer;
 
 public:
 	ErrmsgWindow(const ErrorMessageData &data) : Window(data.HasFace() ? &_errmsg_face_desc : &_errmsg_desc), ErrorMessageData(data)
 	{
 		this->InitNested();
+		this->duration_timer.SetInterval(1000);
```

Comment:
Normally the interval was `3000`. We now change that to `1000`. That means the default of `15 seconds` an error message is visible becomes `5 seconds`. I worry 5 seconds is a bit short to keep it open.

I have been looking if we can fix this cleanly, but as configuration options are not versioned, I am not sure that is possible ..